data_processing/getPathologyAnnotations/config.yaml.template

```diff
@@ -1,3 +1,27 @@
+spark_cluster_config:
```

Comment:
I suggest we use the terms app_config and data_config (previously ingestion_config). So this file name would become app_config.yaml.template to further qualify it as a config for the app.